control-plane-operator/controllers/hostedcontrolplane/kas/config.go

```diff
@@ -206,6 +206,11 @@ func generateConfig(p KubeAPIServerConfigParams, version semver.Version) *kcpv1.
 	args.Set("shutdown-send-retry-after", "true")
 	args.Set("storage-backend", "etcd3")
 	args.Set("storage-media-type", "application/vnd.kubernetes.protobuf")
+	if p.APIServerSTSDirectives == "" {
+		args.Set("strict-transport-security-directives", "max-age=31536000,includeSubDomains,preload")
+	} else {
+		args.Set("strict-transport-security-directives", p.APIServerSTSDirectives)
+	}
```

Comment:
nit: I think it would be cleaner if we set these values in params.go (for both default value and IBM Cloud override). This way it's defined it in one place and we can just simply:
```suggestion
	args.Set("strict-transport-security-directives", p.APIServerSTSDirectives)
```

Reply:
@hasueki I moved things around per your suggestion